src/components/Content/RightPanel/TaskList/TaskList.js

```diff
@@ -253,6 +260,11 @@ class TaskList extends Component {
               autoFocus
               onChange={this.onChangeFilter}
             />
+            <Button className="clear-input"
+                    onClick={ this.clearFilter }
+                    style={{opacity: taskListFilter.length > 0 ? '' : '0', cursor: taskListFilter.length > 0 ? 'pointer' : 'default'}}>
```

Comment:
@4xdk For obscure security reasons, please create 2 CSS classes that you want to choose between here, and select those rather than doing an inline `style`!

Reply:
Sure. Would you have any info on those obscure security reasons? I'm curious now how inline CSS could be a risk